back-end/auth/views.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth import authenticate
+from django.core.exceptions import ObjectDoesNotExist
+from django.views.decorators.csrf import csrf_exempt
+
+from rest_framework.decorators import api_view
+from rest_framework.decorators import permission_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.authtoken.models import Token
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_404_NOT_FOUND,
+)
+
+from drf_yasg.utils import swagger_auto_schema
+
+from auth.swagger import token_auth_header
+
+
+@swagger_auto_schema(method="get", manual_parameters=[token_auth_header])
+@csrf_exempt
+@api_view(["GET"])
+@permission_classes([AllowAny])
+def info(request: Request) -> Response:
+    data = {
+        "roles": ["admin"],
+        "avatar": "mock",
+        "name": "Mock M. M.",
```

Comment:
We are currently mocking user info because it requires token authentication which is not implemented on the front end part yet.